test/output_test_helper.cc

```diff
@@ -134,7 +134,7 @@ class TestReporter : public benchmark::BenchmarkReporter {
   TestReporter(std::vector<benchmark::BenchmarkReporter*> reps)
       : reporters_(reps) {}
 
-  virtual bool ReportContext(const Context& context) {
+  virtual bool ReportContext(const Context& context) BENCHMARK_OVERRIDE {
```

Comment:
Since you are fixing the places that warning flag pointed out in the entire code base, and not just that header,
can you also enable that flag around here https://github.com/google/benchmark/blob/01c0ab7cbb8afdf8af46baed06514f8fae7ee2fb/CMakeLists.txt#L140 ?